scripts/linux/pkg/linux-packaging.md

```diff
@@ -0,0 +1,88 @@
+# RavenDB installation from package on Linux
+
+## What the package does on install
+
+- scatter the files in the default locations
+
+- install systemd service
+
+- add soft links to binaries
+
+- add `ravendb` user and group
+
+- set file permissions properly
+
+- setcap on the Raven.Server
+
+
+## Supported use cases
+
+### Ran under systemd
+The default is when RavenDB is used as a systemd daemon and controlled with systemd.
+The following directory layout (initialized by the package installation process) and permissions is used in this scenario:
+
+#### Filesystem locations and permissions
+
+- settings - `/etc/ravendb/settings.json`
+
+- data - `/var/lib/ravendb/data`
```

Comment:
Default path in `/data/ravendb` ? Should we go for the layout consistency or assume that this is the main purpose of the instance and make the data location obvious?

Reply:
Thinking this over, MySQL & Postgres all store their data in similar paths, so `/var/lib` it is.

Reply:
Yes